misc/start_faux

```diff
@@ -89,6 +89,20 @@ elif [ -n "${options[bacnetfail]}" ]; then
          FauxDeviceEngine.EntryPoint $local_ip $broadcast_ip "Faux-Device-Fail.json"  &
 fi
 
+if [ -n "${options[ntp_client]}" ]; then
+    echo Starting ntp client.
+    java -jar NTPClient/build/libs/NTPClient-1.0-SNAPSHOT.jar "time.google.com" "123" "3" &
+fi
+
+cip_port=41794
```

Comment:
move inside if statement unless they have global applicability

Reply:
Moved 'cip_port=41794' inside the respective if statement.

```
if [ -n "${options[broadcast_client]}" ]; then
    echo Starting broatcast client.
    cip_port=41794
    python TransportClient/client.py $broadcast_ip $cip_port broadcast $duration_seconds $cycle_seconds &
fi
```